dts/arm/st/f7/stm32f7.dtsi

```diff
@@ -634,14 +634,34 @@
 
 		adc1: adc@40012000 {
 			compatible = "st,stm32-adc";
-			reg = <0x40012000 0x400>;
+			reg = <0x40012000 0x50>;
 			clocks = <&rcc STM32_CLOCK_BUS_APB2 0x00000100>;
 			interrupts = <18 0>;
 			status = "disabled";
 			label = "ADC_1";
 			#io-channel-cells = <1>;
 		};
 
+		adc2: adc@40012000 {
```

Comment:
This needs to have the same address as the `reg` node below.
```suggestion
		adc2: adc@40012100 {
```

Reply:
each of the three ADC only has 0x50 as its own address space. however there are three "common registers". luckily the adc_stm32 driver doesn't use these three common registers.

![ADC_regs](https://user-images.githubusercontent.com/151651/106279663-0d43b400-623d-11eb-8f25-bc92d5c27765.png)